views/marts/product/fct_active_users.view.lkml

```diff
@@ -137,34 +193,90 @@ view: fct_active_users {
   dimension: weekly_desktop_active_users {
     type: number
     sql: ${TABLE}.weekly_desktop_active_users ;;
-    label: "Desktop/Client WAU"
-    description: "WAU reported from Rudderstack"
+    label: "Desktop WAU"
+    description: "Desktop WAU reported by telemetry."
   }
 
   measure: total_weekly_desktop_active_users {
     type: sum
     sql: ${weekly_desktop_active_users} ;;
-    label: "Total Desktop/Client WAU"
+    label: "Total Desktop WAU"
+    view_label: "  * Metrics. WAU"
+  }
+
+  dimension: weekly_webapp_active_users {
+    type: number
+    sql: ${TABLE}.weekly_webapp_active_users ;;
```

Comment:
to be consistent with above, why not `weekly_client_webapp_active_users` ; or to simplify, remove client from all?